### PR TITLE
Upgrade Undertow to 2.3.0.Alpha1 in WildFly Preview

### PR DESCRIPTION
**Pull request by ropalka**
[WFLY-16454](https://issues.redhat.com/browse/WFLY-16454) Upgrade Undertow to 2.3.0.Alpha1
[WFLY-15697](https://issues.redhat.com/browse/WFLY-15697) Upgrade Jakarta Server Pages to 3.1.0
[WFLY-15698](https://issues.redhat.com/browse/WFLY-15698) Upgrade Jakarta Servlet API to 6.0.0
[WFLY-15700](https://issues.redhat.com/browse/WFLY-15700) Upgrade Jakarta WebSockets API to 2.1.0

### CODE REVIEW

**Comment by ropalka** [comment]
Just FYI I introduced JspPropertyGroupDescriptorImpl.java in source-transform directory to implement new method introduced in JSP API 3.1.0 @bstansberry .

**Review by bstansberry — APPROVED** [review]

File: ee-9/source-transform/clustering/web/undertow/pom.xml [hunk 1]
```diff
@@ -134,4 +134,39 @@
...
+                                toDelete.forEach {
+                                    def file = new File(project.build.getDirectory() + "/generated-sources/transformed/" + it)
+                                    if (file.isDirectory()) {
+                                        file.deleteDir()
+                                    } else {
+                                        file.delete()
+                                    }
+                                }
+                            </source>
+                        </configuration>
```
Comment (pferraro) [inline ee-9/source-transform/clustering/web/undertow/pom.xml:165]: Clever!

**Review by pferraro — APPROVED** [review]
Thanks @ropalka .

**Comment by bstansberry** [comment]
Thanks @ropalka!

**Pull request merged**